qa/vagrant/src/test/resources/packaging/scripts/20_tar_package.bats

```diff
@@ -115,6 +115,24 @@ setup() {
     export ES_JAVA_OPTS=$es_java_opts
 }
 
+@test "[TAR]" start Elasticsearch with unquoted JSON option {
```

Comment:
We can totally make a new qa project for this. `smoke-test-ingest-disabled` does that.